### ISSUE TITLE
[feature idea] can specify boxart border style per node/edge

### ISSUE THREAD

**Issue by bAndie91** [AlexanderGrooff/mermaid-ascii#24]
as boxart is implemented (#16), it'd be a nice feature to be able to specify the border/edge styles in the diagram source text.
there are several box drawing chars in unicode (below).

specify these node attributes (or other more appropriate in mermaid-js) per node to render them other that the diagram's default:

- weight
  - light
  - heavy
- style
  - solid
  - double
  - double-dash
  - triple-dash
  - quadruple-dash
- corner
  - square
  - round

i have not found example in mermaid-js that edges can be styled with css-like attributes, only by typing the arrows with various chars for lines. we can follow this, although most of the line style are not standardized in mermaid-js, only "normal" (light) and "thick" (i'd translate it to unicode's heavy line, however it looks much more like the double lines) and maybe quadruple-dash looks like dotted.
so one options is to use mermaid-js syntax to change edges line style <https://docs.mermaidchart.com/mermaid/flowchart/edges#edges-with-no-text>

- `-->` - light-solid (default)
- `==>` - heavy-solid
- `##>` - double (not found in mermaid-js)
- `-_->` - double-dash (not found in mermaid-js)
- `-_-_->` - triple-dash (not found in mermaid-js)
- `-.->` - quadruple-dash

# example

```
flowchart LR
node1 ==> node2
style node1 border-weight:heavy,border-style:quadruple-dash
style node2 corner:round
```
would render as:
```
┏┉┉┉┉┉┉┉┓     ╭───────╮
┋       ┋     │       │
┋ node1 ┋════►│ node2 │
┋       ┋     │       │
┗┉┉┉┉┉┉┉┛     ╰───────╯
```

# unicode chars

```
║ U+2551 BOX DRAWINGS DOUBLE VERTICAL

│ U+2502 BOX DRAWINGS LIGHT VERTICAL
┃ U+2503 BOX DRAWINGS HEAVY VERTICAL

╎ U+254E BOX DRAWINGS LIGHT DOUBLE DASH VERTICAL
╏ U+254F BOX DRAWINGS HEAVY DOUBLE DASH VERTICAL

┆ U+2506 BOX DRAWINGS LIGHT TRIPLE DASH VERTICAL
┇ U+2507 BOX DRAWINGS HEAVY TRIPLE DASH VERTICAL

┊ U+250A BOX DRAWINGS LIGHT QUADRUPLE DASH VERTICAL
┋ U+250B BOX DRAWINGS HEAVY QUADRUPLE DASH VERTICAL

╭ U+256D BOX DRAWINGS LIGHT ARC DOWN AND RIGHT
╮ U+256E BOX DRAWINGS LIGHT ARC DOWN AND LEFT
╯ U+256F BOX DRAWINGS LIGHT ARC UP AND LEFT
╰ U+2570 BOX DRAWINGS LIGHT ARC UP AND RIGHT
```

what do you think?

**Comment by AlexanderGrooff** [comment]
I agree, I definitely want to have something like this that implements different 'styles' of arrows, boxes etc. Like you mentioned, there are already syntactical definitions for these different kinds of arrows and I would like to adopt them somehow.

Good ideas!
